script/Ciel/Day 9/09_01_CIEL09_6.txt

```diff
@@ -612,7 +612,8 @@ I'm the only one who has the right to address Shiki."
 -- Page 582, Offset 23625. Mods: @n.
 -- 自分が何であるかを理解しない愚か者に、
 // Front loaded because of how the VA timing triggers work
-My only aim here is to impart the truth to an idiot who doesn't seem to understand what she truly is.
+// I guess we can make it vague
```

Comment:
Submitter comment
>The 'she' here is not maximally well-anchored on Arcueid; I had a hiccup reading it where I went "wait, why is Shiki a girl now?", since the most obvious reference based on sentence-structure-up-to-that-point was that 'she' was the previously-mentioned idiot.

Reply:
Not sure I see the confusion (this dialogue is entirely between Ciel and Arc at this point), but I guess the change is fine